mlir/CMakeLists.txt

```diff
@@ -197,6 +197,7 @@ add_subdirectory(lib/CAPI)
 
 if (MLIR_INCLUDE_TESTS)
   add_definitions(-DMLIR_INCLUDE_TESTS)
+  add_definitions(-DGTEST_NO_LLVM_SUPPORT=0)
```

Comment:
This likely would be best defined inside the unit tests directory?

Add a comment explaining that this is needed because of Wundef